tests/evals/test_pytest.py

```
@@ -0,0 +1,272 @@
+    assert 'Logfire dataset (uppercase): ' in result.stdout
+    assert '\033]8;;https://logfire-us.pydantic.dev/kludex/potato/evals/uppercase\033\\' in result.stdout
+    assert '\033[36mhttps://logfire-us.pydantic.dev/kludex/potato/evals/uppercase\033[0m' in result.stdout
+    assert '\033[0m\033]8;;\033\\\n\nResults' in result.stdout
+    assert 'Logfire evals overview' not in result.stdout
+    assert 'compare?experiment=abc' not in result.stdout
+
+
+def _run_inline_pytest(tmp_path: Path, source: str, *pytest_args: str) -> subprocess.CompletedProcess[str]:
+    test_file = tmp_path / 'test_eval_marker.py'
+    test_file.write_text(textwrap.dedent(source), encoding='utf-8')
+    return subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)